addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -113,5 +113,55 @@ tour.register('edit_link_popover', {
         trigger: '.js_add_menu',
         run: function () {}, // it's a check
     },
+    {
+        content: "Close modal",
+        trigger: '.modal-footer .btn-secondary',
+    },
+    // 3. Test other links (CTA in navbar & links in footer)
+    {
+        content: "Click CTA in navbar",
+        trigger: '#top_menu_container a.btn-primary[href="/contactus"]',
+    },
+    {
+        content: "Popover should be shown (3)",
+        trigger: '.o_edit_menu_popover .o_we_url_link:contains("Contact Us")',
+        run: function () {}, // it's a check
+    },
+    {
+        content: "Click 'Home' link in footer",
+        trigger: 'footer a[href="/"]',
+    },
+    {
+        content: "Popover should be shown (4)",
+        trigger: '.o_edit_menu_popover .o_we_url_link:contains("Home")',
+        run: function () {}, // it's a check
+    },
+    // 4. Popover should close when clicking non-link element
+    {
+        content: "Ensure popover is closed",
+        trigger: 'footer h5',
+    },
+    // 5. Double click shouldn't do anything
+    {
+        content: "Double click on link",
+        trigger: 'html:not(:has(.o_edit_menu_popover))', // popover should be closed
+        run: function () {
+            function createRange(el) {
+                var range = document.createRange();
+                range.selectNodeContents(el);
+                var sel = window.getSelection();
+                sel.removeAllRanges();
+                sel.addRange(range);
+            }
```

Comment:
Creating an inline function to call it only once? No :wink: If this is to "regroup" the code, use a comment instead :wink: Also use `const` instead of `var`. But strange you have to do all that, are you sure it is not because you are not simulating the whole chain of events?

Reply:
>  But strange you have to do all that, are you sure it is not because you are not simulating the whole chain of events?

From what I understand, yes I am sure, you can test the same issue with simply doing `$($0).dblclick()` directly in the browser console.
You will correclty go into the dblclick handler, but the selection & range here https://github.com/odoo-dev/odoo/blame/c8def66e2099d16f61667dd76f43aeda93f8dfe9/addons/web_editor/static/src/js/editor/snippets.editor.js#L3009 will be null.

If you manually double click on a link, the toolbar is shown as it should, but in tours & dblclick in console, it's not.